lib/legacy/polymer.dom.html

```diff
@@ -237,70 +237,61 @@
 
   /**
    * @function
-   * @memberof Polymer
    * @param {boolean=} deep
    * @return {!Node}
    */
-  DomApi.prototype.cloneNode;
+  Polymer.DomApi.prototype.cloneNode;
```

Comment:
I think these now all need to move below the `Polymer.DomApi = DomApi;` line because it's still undefined at this point.

Reply:
whoops! good eyes, i should've seen that 🙈